src/renderer/common/react.ts

```diff
@@ -1,17 +1,17 @@
 import { AnyFunction } from "../../types/util";
-import { byPropsFilter, getByProps, waitFor } from "../modules/webpack";
+import { filters, getModule, waitForModule } from "../modules/webpack";
 
 // @ts-expect-error - webpack exports are loaded async
 let React: typeof import("react") & {
   reactReady: Promise<void>;
   jsx: AnyFunction;
   jsxs: AnyFunction;
 } = {
-  reactReady: waitFor(
-    byPropsFilter(["__SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED", "createElement"]),
+  reactReady: waitForModule(
+    filters.props(["__SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED", "createElement"]),
   ).then((react) => {
     Object.assign(React, react as typeof import("react"));
-    React.jsx = getByProps("jsx")?.jsx as AnyFunction;
+    React.jsx = (getModule(filters.props(["jsx"])) as { jsx: AnyFunction }).jsx;
```

Comment:
Any reason you can't just do
```suggestion
    React.jsx = (getModule(filters.props(["jsx"])) as AnyFunction });
```

Reply:
Unless I'm misreading your suggestion, that wouldn't work, as `getModule(filters.props(["jsx"]))` returns an object with a jsx key.